src/services/userLogin.js

```diff
@@ -0,0 +1,21 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { collection, getDoc } from "firebase/firestore";
+import { auth, db } from "./firebase";
+
+const userLogin = async ({ email, password }) => {
+	try {
+		const userCredential = await signInWithEmailAndPassword(
+			auth,
+			email,
+			password
+		);
+		const userID = userCredential.user.uid;
+		return true;
+		//userID will be used later
+	} catch (err) {
```

Comment:
use Toast or alerts for  showing the error!

Reply:
Hmm...from what I understand generally errors in form input fields are shown on the form itself...I'll add the toast at the end for successful signup and login